packages/tailwindcss-language-server/src/project-locator.ts

```diff
@@ -733,3 +747,9 @@ class FileEntry {
     )
   }
 }
+
+function requiresPreprocessor(filepath: string) {
+  let ext = path.extname(filepath)
+
+  return ext === '.scss' || ext === '.sass' || ext === '.less' || ext === '.styl'
```

Comment:
Do we also want to include `.pcss` or is that not an issue?

Reply:
My thinking was that `.pcss` files (some extension for postcss) the files should still be valid CSS. Buuuut… yeah maybe we should include it anyway. I can relax the check later if need be.